modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/export-form/openExportFormModal.es.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayForm from '@clayui/form';
+import {ClayIconSpriteContext} from '@clayui/icon';
+import ClayModal, {useModal} from '@clayui/modal';
+import {render} from '@liferay/frontend-js-react-web';
+import React, {useState} from 'react';
+import {unmountComponentAtNode} from 'react-dom';
+
+import {ExportFormModalBody} from './ExportFormModalBody.es';
+
+export function ExportFormModal({
```

Comment:
What do you think of putting the entire modal in the previous file and just use this lib to render it? Please, feel free to disagree. 😄

Reply:
Hi @rodrigopaulino,
I tried to make it similar to what was done in [openShareFormModal](https://github.com/carolmariaabb/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/resources/META-INF/resources/admin/js/components/share-form/openShareFormModal.es.js).
Do you think it's okay to keep it different?

Reply:
Let's also wait for Milton's review to answer this one. Depending on his answer, the logic may change.